package-support-policy.md

```diff
@@ -0,0 +1,372 @@
+- [Microsoft.Bcl.HashCode](https://www.nuget.org/packages/Microsoft.Bcl.HashCode) - Its source code used to be hosted in the.NET Core 3.1 branch.
+- [System.Net.WebSockets.WebSocketProtocol](https://www.nuget.org/packages/System.Net.WebSockets.WebSocketProtocol) - Its source code used to be hosted in the .NET 5.0 branch.
+
+## End of support
+
+A package will be supported for as long as it targets frameworks that are still supported. For example: [System.Numerics.Vectors](https://www.nuget.org/packages/System.Json#supportedframeworks-body-tab). Of all the included target frameworks in the package, .NET Standard 2.0 is still in support at the moment of writing this document. As long as there are frameworks compatible with .NET Standard 2.0, the package will remain in support.
+
+## Reporting issues
+
+If you find an issue in an assembly that is hosted in this repo, please report it in our central repo: https://github.com/dotnet/runtime/issues/new/choose.
+
+## Packages
+
+Below is a complete list of packages that we have been published in past releases and an explanation of why they meet (or do not meet) the bar for migration into this repo.
+
+### Supported and migrated
+
+The following packages have been migrated into this repo already and are open for security fixes.
+These packages had a large an complex enough implementation that targets at least one framework that is still in-support.
+
+| Package                                                                                 | Last shipped release |
+|-----------------------------------------------------------------------------------------|----------------------|
+| [Microsoft.Bcl.HashCode](src/Microsoft.Bcl.HashCode/)                                   | 3.1                  |
+| [System.Buffers](src/System.Buffers/)                                                   | 2.1                  |
+| [System.Data.SqlClient](src/System.Data.SqlClient/)                                     | 3.1                  |
+| [System.Json](src/System.Json/)                                                         | 3.1                  |
+| [System.Memory](src/System.Memory/)                                                     | 2.1                  |
+| [System.Net.WebSockets.WebSocketProtocol](src/System.Net.WebSockets.WebSocketProtocol/) | 5.0                  |
+| [System.Numerics.Vectors](src/System.Numerics.Vectors/)                                 | 2.1                  |
+| [System.Reflection.DispatchProxy](src/System.Reflection.DispatchProxy/)                 | 3.1                  |
+| [System.Runtime.CompilerServices.Unsafe](src/System.Runtime.CompilerServices.Unsafe/)   | 6.0                  |
+| [System.Threading.Tasks.Extensions](src/System.Threading.Tasks.Extensions/)             | 2.1                  |
+| [System.Xml.XPath.XmlDocument](src/System.Xml.XPath.XmlDocument/)                       | 1.1                  |
+
+### Implementation too simple
```

Comment:
These tables show how you've decided whether or not to include a package in this repo, but that doesn't really answer _customer_ questions they might have about a package.  It's also difficult to navigate all these.

Here are some customer questions I'd hope we could answer with a table.
1. Should I still reference package X on framework Y?  (Maybe we could have this for each minium supported framework and NETStandard)
2. Why isn't there a new version of package X?

Maybe we could have a single table that covers this?

Reply:
From our chat today, I summarized the explanation:
If we had a single table that listed every package that we ever shipped or that we would expect to be covered by maintenance-packages, for the ones that are still in service we say update to latest, for the others we can tell folks not to reference it anymore, or yes you can reference and it does not need to change.